instrumentation/jdbc/javaagent/src/main/java/io/opentelemetry/javaagent/instrumentation/jdbc/JdbcUtils.java

```diff
@@ -69,8 +74,27 @@ public static Connection connectionFromStatement(Statement statement) {
     return connection;
   }
 
+  public static SqlStatementInfo sanitizeAndExtractInfo(String sql) {
+    try {
+      return sqlToStatementInfoCache.get(
+          sql,
+          () -> {
+            log.debug("SQL statement cache miss");
+            try {
+              return SqlStatementInfoExtractor.extract(sanitizeSql(sql));
+            } catch (ParseException e) {
+              log.debug("Could not extract sql info", e);
+              return new SqlStatementInfo(null, null, null);
+            }
+          });
+    } catch (ExecutionException e) {
+      log.info("Sql statement cache error", e);
```

Comment:
Hmm, should we really log it as info? Users will see this exception (if it ever happens...) with the default javaagent logging.

Reply:
I chose `info` because the docs indicate that it is unusual for this to happen, and is usually related to an underlying `InterruptedException`.  I wouldn't think the user would see this much, but if it's happening a lot it would be good to know wouldn't it? 

I don't feel strongly, and I could back it down to debug.